src/common/enums.js

```diff
@@ -221,3 +221,14 @@ export const statusyMessage = {
     class: 'text-blue cursor-pointer noMargin',
   },
 };
+
+// Coordinator release state label colors
+export const stateColors = {
```

Comment:
Maybe `releaseStateColors` would be more descriptive to differentiate from document states.